README.md

```diff
@@ -165,6 +165,7 @@ These are some of the editor's favorite submissions:
 
 [//]: # (Non-solution URLs)
 [php-soln]:  https://github.com/eatnumber1/goal/tree/master/incomplete/php
+[php-solv]:  https://github.com/eatnumber1/goal/tree/master/solved/php
```

Comment:
If this is a solution, it shouldn't be under the `Non-solution URLs` section. Can you move it up?